runelite-client/src/main/java/net/runelite/client/plugins/party/PartyPlugin.java

```diff
@@ -145,9 +149,8 @@ public class PartyPlugin extends Plugin
 	private PartyPanel panel;
 	private NavigationButton navButton;
 
-	private int lastHp, lastPray;
-	private String lastCharacterName = "";
 	private WorldPoint lastLocation;
+	private StatusUpdate lastStatus;
```

Comment:
this probably needs to be nulled at shutdown or else the plugin doesn't restart correctly